packages/prettier-config/index.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "arrowParens": "avoid",
+  "bracketSpacing": false,
+  "endOfLine": "lf",
```

Comment:
https://prettier.io/docs/en/options.html#end-of-line

are we OK with `lf`?

Reply:
Yes